src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py

```diff
@@ -43,6 +43,35 @@
     MIN_DEPTH_REPLACEMENT_VALUE,
 )
 
+CLASSES = [
```

Comment:
This should be defined in some constants.py file in habitat_uncertainty and only imported in this file - instead of redefining completely. Becaause you also might use this list somewhere else and then there could be differences

Reply:
Yes, I'll add this to a new constants.py and import them on all other files